deployment/CP05MOAS-GL388_Deploy.csv

```diff
@@ -77,9 +77,9 @@ AR66,,AR66,,CP05MOAS-GL388-02-FLORTM000,13,1,2022-04-10T16:00:00,2022-04-10T22:5
 AR66,,AR66,,CP05MOAS-GL388-03-CTDGVM000,13,1,2022-04-10T16:00:00,2022-04-10T22:54:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00388,CGINS-CTDGVM-09363,39.93303,-70.58503,,1000,1000,FZ-1,,
 AR66,,AR66,,CP05MOAS-GL388-04-DOSTAM000,13,1,2022-04-10T16:00:00,2022-04-10T22:54:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00388,CGINS-DOSTAM-00418,39.93303,-70.58503,,1000,1000,FZ-1,,
 AR66,,AR66,,CP05MOAS-GL388-05-PARADM000,13,1,2022-04-10T16:00:00,2022-04-10T22:54:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00388,CGINS-PARADM-50238,39.93303,-70.58503,,1000,1000,FZ-1,,
-AR66,,,,CP05MOAS-GL388-00-ENG000000,14,1,2022-04-11T16:24:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00388,CGCON-GLDRCP-00388,39.9338,-70.66867,,1000,1000,FZ-1,,
-AR66,,,,CP05MOAS-GL388-01-ADCPAM000,14,1,2022-04-11T16:24:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00388,CGINS-ADCPAM-43106,39.9338,-70.66867,,1000,1000,FZ-1,,
-AR66,,,,CP05MOAS-GL388-02-FLORTM000,14,1,2022-04-11T16:24:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00388,CGINS-FLORTM-04467,39.9338,-70.66867,,1000,1000,FZ-1,,
-AR66,,,,CP05MOAS-GL388-03-CTDGVM000,14,1,2022-04-11T16:24:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00388,CGINS-CTDGVM-09363,39.9338,-70.66867,,1000,1000,FZ-1,,
-AR66,,,,CP05MOAS-GL388-04-DOSTAM000,14,1,2022-04-11T16:24:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00388,CGINS-DOSTAM-00418,39.9338,-70.66867,,1000,1000,FZ-1,,
-AR66,,,,CP05MOAS-GL388-05-PARADM000,14,1,2022-04-11T16:24:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00388,CGINS-PARADM-50238,39.9338,-70.66867,,1000,1000,FZ-1,,
\ No newline at end of file
+AR66,,,,CP05MOAS-GL388-00-ENG000000,14,1,2022-04-11T16:24:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00388,CGCON-GLDRCP-00388,39.9338,-70.66867,,1000,1000,SS-1,,
+AR66,,,,CP05MOAS-GL388-01-ADCPAM000,14,1,2022-04-11T16:24:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00388,CGINS-ADCPAM-43106,39.9338,-70.66867,,1000,1000,SS-1,,
+AR66,,,,CP05MOAS-GL388-02-FLORTM000,14,1,2022-04-11T16:24:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00388,CGINS-FLORTM-04467,39.9338,-70.66867,,1000,1000,SS-1,,
+AR66,,,,CP05MOAS-GL388-03-CTDGVM000,14,1,2022-04-11T16:24:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00388,CGINS-CTDGVM-09363,39.9338,-70.66867,,1000,1000,SS-1,,
+AR66,,,,CP05MOAS-GL388-04-DOSTAM000,14,1,2022-04-11T16:24:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00388,CGINS-DOSTAM-00418,39.9338,-70.66867,,1000,1000,SS-1,,
+AR66,,,,CP05MOAS-GL388-05-PARADM000,14,1,2022-04-11T16:24:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00388,CGINS-PARADM-50238,39.9338,-70.66867,,1000,1000,SS-1,,
```

Comment:
good - moved to SS-1 as noted in slack channel

Reply:
Looks good.